accepted/global-json-updates.md

```diff
@@ -0,0 +1,790 @@
+# Proposal for global.json updates for preview and roll-forward
+
+---
+**NOTE:** 
+
+SDK version numbers do not use semantic versioning. [Docs has an article on SDK versioning](https://docs.microsoft.com/en-us/dotnet/core/versions/).
+
+---
+
+`global.json` allows users to pin the version of the SDK they are using. This was added to .NET Core as a way for users to roll-back to a previous version per project until a fix could be provided if there a problem occurred. While our guidance remains to use it only for special circumstances, users now use it for numerous reasons. 
+
+This document describes changes to `global.json` to provide more flexibility to reflect the wider usage. 
+
+## Definitions
+
+The term *SDK patch* is used to describe the last two positions of the SDK major/minor/feature/patch (n in x.y.znn).
+
+The term *feature band* is used to describe the SDK major/minor/feature/patch (x.y.z in x.y.znn).
+
+Given x.y.znn
+
+* x is the major version
+* y is the minor version
+* z is the feature band
+* nn is the patch
+
+## Current behavior
+
+The SDK resolvers (in both the CLI and VS) search from the current location (project in VS) up the directory hierarchy until they encounter a `global.json` file. When the resolver finds one, _it stop looking_ and attempts to resolve the SDK version specified in that file. This happens even if this `global.json` does not specify an SDK version.
```

Comment:
This paragraph aims to define the experience in terms of both the CLI and VS, but then defined the current location in terms of VS. I assume current location is equivalent to CWD. Is that right?